fusor-ember-cli/app/templates/components/text-f.hbs

```diff
@@ -4,7 +4,7 @@
                      value=value
                      placeholder=placeholder
                      type=typeInput
-                     key-up="showValidationErrorsKeyUp"
+                     key-press="showValidationErrorsKeyUp"
```

Comment:
@isratrade I had to go look it up (http://www.quirksmode.org/dom/events/keys.html if you're interested). When testing it out, I actually had the best luck with key-down.  key-press didn't trigger when I used backspace to blank out values.  It looks like it's triggered on inserting test, but I guess not deleting it. key-up didn't trigger when holding down a key.

key-down was triggered for all the use cases I could think of.  We should probably change the action name to match whatever we choose.